qiskit_nature/properties/second_quantization/electronic/angular_momentum.py

```diff
@@ -62,7 +60,10 @@ def from_legacy_driver_result(cls, result: LegacyDriverResult) -> "AngularMoment
         Raises:
             QiskitNatureError: if a WatsonHamiltonian is provided.
         """
-        cls._validate_input_type(result, LegacyElectronicStructureDriverResult)
+        # pylint: disable=import-outside-toplevel
+        from qiskit_nature.drivers import QMolecule
```

Comment:
I need to double check whether the "outside-toplevel" imports are still necessary (I was running into some cyclic import issues before).
In any case, I still need to wrap the import in all occurrences with filters for the `DeprecationWarning`s

Reply:
This is indeed no longer necessary